docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Infinispan marshalling changes
+
+Marshalling is the process of converting java objects into bytes to send them across the network between {project_name} server.
+With {project_name} 26, the marshalling library has changed from JBoss Marshalling to Infinispan Protostream.
+The libraries are not compatible between each other and, it requires some steps to ensure the session data is not lost.
+
+WARNING: JBoss Marshalling and Infinispan Protostream are not compatible betwen each other and incorrect usage may lead to data loss.
+
+If the loss of session is acceptable for your deployment, you may skip this section.
+
+To prevent losing sessions, it is required to upgrade to {project_name} 25 first and enable the feature `persistent-user-session`.
+To enable this feature use the following command:
+
+[source,bash]
+----
+bin/kc.sh build --features=persistent-user-session ...
+----
+
+This process will import all the session from in-memory Infinispan caches into the database.
```

Comment:
Should we be using the `{jdgserver_name}` placeholder here, and in your other docs?

Reply:
I replaced in this line but left `Infinispan Protostream` unchanged since "protostream" is not a product so the name stays the same.